docs/developer.md

```diff
@@ -1,3 +1,23 @@
 # blob-uploader.js
 
-Coming soon...
+This library wraps the blob uploader service [Blob uploader](https://github.com/blinkmobile/blob-uploader) and AWS S3 to allow uploading and retrieve of blobs from S3 without having to work with AWS credentials and the AWS SDK directly.
+## Usage
+1.  Initialise the blobUploader with the URL of the blob uploader service, e.g. 
+```
+new blobUploader('https://bm-blob-uploader-dev.api.blinkm.io/')
```

Comment:
Because this is a contructor, I would prefer it was `BlobUploader` with a capital. Thoughts?